### PR TITLE
change allowIssueReporting default to true

### PR DESCRIPTION
**Pull request by mwest1066**
I also changed the wording of the student interface to emphasize "error" rather than "issue":
assessment-report2

### CODE REVIEW

File: doc/assessment.md [hunk 1]
```diff
@@ -108,9 +107,9 @@ By default Exam assessments will auto-close after six hours of inactivity by the
 
 ## Issue reporting
 
-To allow students to report issues with questions (incorrect answers, unclear wording, etc), set the `"allowIssueReporting": true` property in the `infoAssessment.json` file. This option defaults to whatever is set for the course instance, or false if nothing is set.
+To allow students to report issues with questions (incorrect answers, unclear wording, etc), set the `"allowIssueReporting": false` property in the `infoAssessment.json` file. This option defaults to whatever is set for the course instance, or true if nothing is set.
```
Comment (nwalters512) [inline doc/assessment.md]: This wording seems wrong.

File: pages/partials/questionScorePanel.ejs [hunk 2]
```diff
@@ -34,18 +34,28 @@
...
     <button class="btn btn-xs btn-secondary" type="button" data-toggle="collapse" data-target="#issueCollapse" aria-expanded="false" aria-controls="issueCollapse">
-      Report an issue with this question
+      Report an error in this question
     </button>
     <div class="collapse" id="issueCollapse">
       <hr />
       <form method="POST">
+        <p class="small">
+          This form is only for reporting errors in the question
+          itself. Do not use this form if you just don't know how to
```
Comment (nwalters512) [inline pages/partials/questionScorePanel.ejs]: HTML entities make syntax highlighting better here 🙂

File: doc/assessment.md (hunk 1 above)
Comment (mwest1066) [reply doc/assessment.md]: Fixed in new PR.

File: pages/partials/questionScorePanel.ejs (hunk 2 above)
Comment (mwest1066) [reply pages/partials/questionScorePanel.ejs]: Ewww! I wish they'd fix their highligher :-)

**Pull request merged**